packages/gatsby/lib/redux/plugin-runner.js

```diff
@@ -13,4 +13,9 @@ store.subscribe(() => {
     const node = state.nodes[state.lastAction.payload.id]
     apiRunnerNode(`onNodeCreate`, { node })
   }
+
+  if (state.lastAction.type === "UPSERT_PAGE") {
+    const page = state.nodes[state.lastAction.payload]
+    apiRunnerNode(`onUpsertPage`, { page })
```

Comment:
Fixed in https://github.com/gatsbyjs/gatsby/commit/ec011f400861e961e537be7e0669c4b5d65aa0e3

Published 1.0.0-alpha13-alpha.ec011f40 w/ fix